WORKSPACE

```diff
@@ -0,0 +1,25 @@
+workspace(name = "autocxx.rs")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_rust",
+    sha256 = "aaaa4b9591a5dad8d8907ae2dbe6e0eb49e6314946ce4c7149241648e56a1277",
+    urls = ["https://github.com/bazelbuild/rules_rust/releases/download/0.16.1/rules_rust-v0.16.1.tar.gz"],
+)
+
+load("@rules_rust//rust:repositories.bzl", "rules_rust_dependencies", "rust_register_toolchains")
+
+rules_rust_dependencies()
+
+rust_register_toolchains(
+    versions = ["1.66.0"],
```

Comment:
Is this dependent on the CI environment?

Reply:
Ah, it is not necessary. It should be alright for `stable` and `nigthly` which will be registered as per http://bazelbuild.github.io/rules_rust/.